### ISSUE TITLE
[BUG] Getting logged out of the system 

### ISSUE THREAD

**Issue by fossdot** [fossunited/fossunited#350]
### Describe the bug

I am getting logged out of the system in every 5-10 minutes.
User type: System User

**Comment by fossdot** [comment]
I noticed a pattern: whenever I'm trying a create a new record in a doctype, I'm getting logged out.

**Comment by mngshm** [comment]
I'm inspecting this issue and also talking with the support team at Frappe. Additionally, will get in contact with someone personally for discussing about this.

**Comment by harshtandiya** [comment]
@Mangeshrex any update on this?

**Comment by mngshm** [comment]
A couple of updates regarding this issue: 

This logout problem only happens on Firefox (tested on Firefox, Brave, Chromium, Chrome) 

1. We had a discussion with Ankush from Frappe on the their helpdesk and tried a couple of things including removing the cloudflare DNS and testing if the issue persists on https://staging-fossunited.frappe.cloud. I did the tests and yes the issue still persists
2. There was a suspect that cloudflare cdn could be the one (another of ankush's hunch), which we were using while fetching the fontawesome icons through their CDN. I tried the code (which was also not being used) in a recent [PR](https://github.com/fossunited/fossunited/pull/384) and that did not improve the situation either.
3. Apart from all of this another doubt was with set_user method getting used in frappe framework and that neither has anything to do with this long running issue as it has been used in the razorpay integration and the this issue happens moreover on through desk and frontend. 

Some more process readings from a report about browser: 

After reading some reports I found that browsers use navigational tracking to keep track or identify whether the users are same on one site and the other. and since Firefox is deleting cookies as soon as some kind of action is triggered, the user gets logged out. hence, if I'm not wrong it does not get any opportunity to create a [bounce tracking record](https://www.fastcompany.com/90663878/bounce-tracking-privacy-browsers-brave-firefox-safari-edge). 

source: https://privacycg.github.io/nav-tracking-mitigations/ 

I'll try talking with ankush  or someone at frappe and maybe setup a call, to try tackle it faster with their help.
